docs/source/contributing/developing.rst

```diff
@@ -5,87 +5,125 @@ Developing for Dissect
 
     Thinking about writing your own Dissect module? Awesome! We kindly ask that you use the ``dissect.contrib.*`` namespace!
 
-Development
------------
 
-Dissect is build and tested against Python 3.9 (CPython and PyPy). Older versions may not work, as features are used which
+
+We very much welcome contributions from the community that improve Dissect. Before making your first contribution, we
+ask you to read our documentation on the development process, style guide and tooling carefully as they will help you make the contributing process as easy as possible.
+
+These pages are organized as follows:
+
+1. `Development process`_: information regarding the development process, including branching, review process and expectations on
+testing.
+
+2. :doc:`Style guide </contributing/style-guide>`: information on how to style code and documentation for a uniform style across all Dissect projects.
+
+3. :doc:`Tooling </contributing/tooling>`: information regarding the available tooling for building and testing code and documentation.
+
+
+Development process
+-------------------
+
+
+Python and tox versions
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Dissect is built and tested against Python 3.9 (CPython and PyPy). Older versions may not work, as features are used which
```

Comment:
```suggestion
Dissect is built and tested against Python 3.9, 3.10 and 3.11 (CPython and PyPy). Older versions may not work, as features are used which
```
We also test against 3.10 and 3.11 these days.

Reply:
But not PyPy, that is only 3.9.

Reply:
Updated the suggestion a bit to reflect that, but feel free to just completely rewrite it however you see fit @martinvanhensbergen :)

Reply:
I have changed this section somewhat; please check again.

Reply:
LGTM